### PR TITLE
Windows WMI fix: make Description *string

### PR DESCRIPTION
**Pull request by requilence**
WMI returns nil Description for some services

### CODE REVIEW

File: pkg/monitoring/services/services_windows.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ var ErrorNotImplementedForOS error
 type Win32_Service struct {
 	Name             string
 	DisplayName      string
-	Description      string
+	Description      *string
```
Comment (soupdiver) [inline pkg/monitoring/services/services_windows.go]: Why do we have to make it a pointer?

File: pkg/monitoring/services/services_windows.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func ListServices(autostartOnly bool) (map[string]interface{}, error) {
 	defer cancel()
 
 	var wmiServices []Win32_Service
-	err := wmiQueryWithContext(ctx, "Select Name,DisplayName,Description,StartMode,State,State,DelayedAutoStart from Win32_Service", &wmiServices)
+	err := wmiQueryWithContext(ctx, "Select Name,DisplayName,Description,StartMode,State,Status,DelayedAutoStart from Win32_Service", &wmiServices)
```
Comment (soupdiver) [inline pkg/monitoring/services/services_windows.go]: Why change State => Status?

File: pkg/monitoring/services/services_windows.go (hunk 1 above)
Comment (requilence) [reply pkg/monitoring/services/services_windows.go]: because of this
image

File: pkg/monitoring/services/services_windows.go (hunk 2 above)
Comment (requilence) [reply pkg/monitoring/services/services_windows.go]: it was a typo

**Pull request merged**